src/ol/View.js

```diff
@@ -1448,6 +1463,21 @@ class View extends BaseObject {
    * @api
    */
   endInteraction(opt_duration, opt_resolutionDirection, opt_anchor) {
+    let anchor = opt_anchor;
+    if (anchor) {
+      anchor = fromUserCoordinate(anchor, this.getProjection());
```

Comment:
Same as for `beginInteraction`.

Reply:
5d4e77151c3eb2c8cd509969a291f4d6c9e3402c